requirements/dev.in

```diff
@@ -0,0 +1,3 @@
+-r tests.in
```

Comment:
While I'm a fan of pip tools for applications, for libraries we normally use `tox` on `pytest-dev`, so please remove all requirements files.

Reply:
done.